### PR TITLE
Fix for delayed loader not working properly.

### LINKED ISSUE #1328: Delayed loading displays even if it should not
### Steps to reproduce

1. Open a room that has already been opened, or a room that you are opening for the first time with  few members.
2. immediately tap on the avatar of a user

#### What did you expect?
Since the members should already been cached in the store, the get member function is very fast, and the loading since is delayed should not appear at all.
#### What happened instead?
The loading the displays, even if the call was actually very fast, so fast that in fact that the the other view has already been pushed in the meantime, however then the loading displays.

Here is a video:
https://github.com/vector-im/element-x-ios/assets/34335419/94da4a61-8f4b-4151-a8ff-92eb6a89fa92

### Your phone model

_No response_

### Operating system version

_No response_

### Application version

_No response_

### Homeserver

_No response_

### Will you send logs?

No

### PR DESCRIPTION
**Pull request by Velin92**
fixes #1328

### CODE REVIEW

File: ElementX.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -3615,14 +3618,6 @@
...
 			sourceTree = "<group>";
 		};
-		"TEMP_FFE5FDBA-B4DD-4FF7-B172-18026F248E20" /* element-x-ios */ = {
-			isa = PBXGroup;
-			children = (
-				41553551C55AD59885840F0E /* secrets.xcconfig */,
-			);
-			path = "element-x-ios";
-			sourceTree = "<group>";
-		};
```
Comment (pixlwave) [inline ElementX.xcodeproj/project.pbxproj:3625]: @alfogrillo Does this mean the git hook to ignore the file is ignoring the checkout too?

File: ElementX/Sources/Other/Extensions/Duration.swift [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+//
+
+import Foundation
+
+extension Duration {
+    /// Use this to to convert `Duration` in seconds as a `Double`.
+    /// `components.seconds` is not reliable because its type is `Int64`.
+    var seconds: Double {
+        let v = components
+        return Double(v.seconds) + Double(v.attoseconds) * 1e-18
```
Comment (pixlwave) [inline ElementX/Sources/Other/Extensions/Duration.swift:24]: How come not this?
```suggestion
        Double(components.seconds) + Double(components.attoseconds) * 1e-18
```

File: ElementX/Sources/Other/UserIndicator/UserIndicatorController.swift [hunk 3]
```diff
@@ -58,7 +58,7 @@ class UserIndicatorController: ObservableObject, UserIndicatorControllerProtocol
         } else {
             if let delay {
                 delayedIndicators.insert(indicator.id)
-                Timer.scheduledTimer(withTimeInterval: Double(delay.components.seconds), repeats: false) { [weak self] _ in
+                Timer.scheduledTimer(withTimeInterval: Double(delay.seconds), repeats: false) { [weak self] _ in
```
Comment (pixlwave) [inline ElementX/Sources/Other/UserIndicator/UserIndicatorController.swift:61]:
```suggestion
                Timer.scheduledTimer(withTimeInterval: delay.seconds, repeats: false) { [weak self] _ in
```

**Review by pixlwave — APPROVED** [review]
LGTM (but should double check about that removal of the secrets config).

**Review by alfogrillo — APPROVED** [review]
LGTM

File: ElementX.xcodeproj/project.pbxproj (hunk 1 above)
Comment (alfogrillo) [reply ElementX.xcodeproj/project.pbxproj:3625]: Mmmh this looks wrong.
@Velin92 can you check if targets have been configured like this?

Screenshot 2023-07-14 alle 16 09 31

If not, please generate the project again.

Comment (Velin92) [reply ElementX.xcodeproj/project.pbxproj:3625]: now they should be, just pushed again

**Pull request merged**